custom_components/tuya_local/devices/kogan_lx10_vacuum.yaml

```diff
@@ -6,15 +6,19 @@ primary_entity:
   entity: vacuum
   dps:
     - id: 101
-      name: power # clean_switch
+      name: start # clean_switch
       type: boolean
     - id: 102
-      name: activate # pause_switch
+      name: pause # pause_switch
       type: boolean
     - id: 104
       name: command # clean_mode
       type: string
       mapping:
+        - value: start
```

Comment:
I never tried using a value in a mapping without a dps_val before.  I am not sure if it will work without side effect, as it makes it the default mapping (and there are two, which makes it indeterminate which is default).